packages/kit/src/exports/index.js

```diff
@@ -84,6 +84,14 @@ export function isHttpError(e, status) {
 /**
  * Redirect a request. When called during request handling, SvelteKit will return a redirect response.
  * Make sure you're not catching the thrown redirect, which would prevent SvelteKit from handling it.
+ *
+ * Most common status codes:
+ *  * `303 See Other`: redirect as a `GET` request
```

Comment:
Is 303 actually common? I can't say I've seen it used much

Reply:
It's the best code to use when redirecting after a POST request (e.g. login form) because it enforces the redirect to be performed as a GET request.

Reply:
```suggestion
 *  * `303 See Other`: redirect as a `GET` request (often used after a form POST request)
```